back/spec/models/notifications/voting_basket_not_submitted_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Notifications::VotingBasketNotSubmitted do
+  describe 'make_notifications_on' do
+    it 'makes a notification when a basket is flagged as not submitted' do
```

Comment:
I think it could be useful to test some edge cases here, like when the phase is already passed.